db-tests/src/test/java/TestTriggers.java

```diff
@@ -1067,5 +1068,105 @@ SELECT COUNT(*)
         }
       }
     }
+
+    @Test
+    void shouldCancelOnMissionModelUpdate() throws SQLException {
+      try (final var statement = connection.createStatement()) {
+        statement
+            .executeUpdate(
+                """
+                    UPDATE mission_model
+                    SET name = 'updated-name-%s'
+                    WHERE id = %s;"""
+                    .formatted(UUID.randomUUID().toString(), missionModelId)
+            );
+
+        try (final var res = statement.executeQuery(
+            """
+                SELECT canceled
+                FROM simulation_dataset
+                WHERE simulation_id = %s;"""
+                .formatted(simulationDatasetId.simulation_id())
+        )) {
+          res.next();
+          assertTrue(res.getBoolean("canceled"));
```

Comment:
Hmmm. Should it be verified at the start of the test that canceled is false too, so that we know canceled was actually set to true by the update? This applies to the tests below as well.

Reply:
Did this on all these tests